chapters/data-collection.tex

```diff
@@ -1,140 +1,97 @@
+and establish and document the research team's right to use the data. 
 Except in the case of primary surveys funded by the research team, 
 the data is typically not owned by the research team. 
-Data ownership and licensing agreements are required 
-for the research team to access the data and publish derivative work. 
-
-
-\subsection{Data ownership}
-
-Data ownership is a tricky subject,
-as many jurisdictions have differing laws regarding data and information,
-and you may even be subject to multiple conflicting regulations.
-In some places, data is implicitly owned by the people who it is about.
-In others, it is owned by the people who collected it.
-In still more, it is highly unclear and there may be varying norms.
-The best approach is always to consult with a local partner
-and to make explicit agreements (including consent, where applicable)
-about who owns any data that is collected.
-Particularly where personal data, business data, or government data is involved
--- that is, when people are disclosing information to you
-that you could not obtain simply by walking around and looking --
-you should be extremely clear up front about what will be done with data
-so that there is no possibility of confusion down the line.
-
-As with all forms of ownership,
-data ownership can include a variety of rights and responsibilities
-which can be handled together or separately.
-Ownership of data may or may not give you
-the right to share that data with other people,
-the right to publish or reproduce that data,
-or even the right to store data in certain ways or in certain locations.
-Again, clarity is key.
-If you are not collecting the data directly yourself --
-for example, if a government, company, or agency is doing it for you --
-make sure that you have an explicit agreement with them
-about who owns the resulting data
-and what the rights and responsibilities of the data collectors are,
-including reuse, storage, and retention or destruction of data.
-
-\subsection{Data licensing agreements}
-
-Data licensing is the formal act of giving some data rights to others
-while retaining ownership of a particular dataset.
-If you are not the owner of the dataset you want to analyze,
-you should enter into a licensing or terms-of-use agreement to access it for research purposes.
-Similarly, when you own a dataset,
-you must consider whether the data can be made accessible to other researchers,
-and what terms-of-use you require. 
-
-As a researcher, it is your responsibility to respect the rights
-of people who own data and people who are described in it;
-but it is also your responsibility to make sure
-that information is as available and accessible it can be.
+Rather, they must enter into data licensing agreements 
+to access the data and publish derivative work. 
+These agreements should make clear from the outset whether the 
+research team can make the original data, or any portion thereof, public. 
```

Comment:
```suggestion
research team can make the original data, any portion thereof, or derivatives\sidenote{
  \textbf{Derivatives} of a data can be indicators, aggregates etc. based on the data.} 
 of the data, public. 
```
Don't know if this is the best way to do it, but I think we should include derivatives already here.